roles/edpm_update/tasks/containers.yml

```diff
@@ -0,0 +1,105 @@
+---
+
+- name: Login to container registries if needed
```

Comment:
i think all of these also need become:true

Reply:
actually you can't use become:true on include_role, so maybe just set it in the update.yml playbook. Either that or we'll need to go through and add it on every task that needs it.

Reply:
when I tested this, I used become:true in the playbook, so let's just go with that for now.

Reply:
Actually, we can!

https://docs.ansible.com/ansible/latest/collections/ansible/builtin/include_role_module.html#parameter-apply
e.g.

```
- name: Login to container registries if needed
  ansible.builtin.include_role:
    name: osp.edpm.edpm_podman
    tasks_from: login.yml
    apply:
      become: true
  tags:
    - edpm_podman
    - edpm_update
```